lib/config.js

```diff
@@ -70,7 +70,11 @@ exports.load = function(fileName, currentEnv) {
   } else if(config['default']) {
     out.setCurrent(config['default']);
   } else if(config.env) {
-    out.setCurrent(config.env);
+    if (config.env.ENV) {
```

Comment:
This can be quite useful, however I would maybe rename the variable `env`, as this might be confusing. Maybe renaming it to `default_env`.